lib/default.nix

```diff
@@ -65,7 +65,7 @@ let
       lessThan listToAttrs pathExists readFile replaceStrings seq
       stringLength sub substring tail;
     inherit (self.trivial) id const pipe concat or and bitAnd bitOr bitXor
-      bitNot boolToString mergeAttrs flip mapNullable inNixShell min max
+      bitNot boolToString mergeAttrs flip mapNullable inNixShell isFloat min max
```

Comment:
how was it exposed before?

Reply:
`lib.trivial` only, unlike the other three `lib.trivial.is*`